examples/dmenu-example.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+#
+# Requirements: jq, notify-send
+#
+# If you use dmenu, consider applying this patch and setting MENU='dmenu -P'
+#     https://tools.suckless.org/dmenu/patches/password/
+#
+
+# Set dmenu program and its options.
+# You can use wmenu -P for Wayland.
+MENU='dmenu'
```

Comment:
Since polkit is a component for controlling system-wide privileges, making it secure is important.

I rather have a failing example than having a functioning, but unsecure one. So, I prefer the default to assume the dmenu patch is applied
```suggestion
MENU='dmenu -P'
```

Reply:
Done!